ShopApp/App/Modules/Shared/Views/Cells/TableView/QuantityDropDownTableViewCell/QuantityDropDownTableViewCell.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  QuantityDropDownTableViewCell.swift
+//  ShopApp
+//
+//  Created by Evgeniy Antonov on 4/23/18.
+//  Copyright © 2018 RubyGarage. All rights reserved.
+//
+
+import DropDown
+
+class QuantityDropDownTableViewCell: DropDownCell {
+    private let fontSize: CGFloat = 14
+    
+    public func configure(with textCenter: Bool) {
+        optionLabel.textAlignment = textCenter ? .center : .left
```

Comment:
Add tests for this block

Reply:
It's subclass of pod's class. I can't find way to generate this cell for tests